ovos_adapt/intent.py

```
@@ -16,8 +16,7 @@
 __author__ = 'seanfitz'
 
 import itertools
-
-CLIENT_ENTITY_NAME = 'Client'
+from ovos_workshop.intents import Intent, IntentBuilder
```

Ruff

`ovos_workshop.intents.IntentBuilder` imported but unused

Remove unused import: `ovos_workshop.intents.IntentBuilder`

(F401)